src/js/tech/html5.js

```diff
@@ -437,7 +437,13 @@ class Html5 extends Tech {
    * @method play
    */
   play() {
-    this.el_.play();
+    const playPromise = this.el_.play();
+
+    // Catch/silence error when a pause interrupts a play request
+    // on browsers which return a promise
+    if (playPromise !== undefined) {
+      playPromise.catch((e) => {});
```

Comment:
`catch` might be an issue on IE8.
Also, we should probably verify this is a promise before proceeding.